batch/device-manager/src/app/interceptors/ErrorKeycloakHandler.interceptor.ts

```diff
@@ -0,0 +1,10 @@
+import { NextFunction, Request, Response } from 'express'
+
+export abstract class ErrorKeycloakHandlerInterceptor {
```

Comment:
Todos os erros da aplicação deveriam ser tratados no mesmo local para inibir a complexidade e evitar chamadas inúteis.